### PR TITLE
Update everything to AndroidX

### PR DESCRIPTION
**Pull request by mario**
Please note that Jetifier is still turned on due to the ViewPager transformation library, but I do plan to tackle that in a future PR. 

Also, this PR depends on the two of my previous PRs.

### CODE REVIEW

**Comment by vanniktech** [comment]
@rubengees this is no longer blocked

**Comment by mario** [comment]
I'll do a rebase next week.

**Comment by mario** [comment]
This is now ready for merging.

File: app/src/main/java/com/vanniktech/emoji/sample/MainActivity.java [hunk 1]
```diff
@@ -62,7 +62,7 @@
 
     final RecyclerView recyclerView = findViewById(R.id.main_activity_recycler_view);
     recyclerView.setAdapter(chatAdapter);
-    recyclerView.setLayoutManager(new LinearLayoutManager(this, LinearLayoutManager.VERTICAL, false));
+    recyclerView.setLayoutManager(new LinearLayoutManager(this, RecyclerView.VERTICAL, false));
```
Comment (rubengees) [inline app/src/main/java/com/vanniktech/emoji/sample/MainActivity.java]: Is this right?

Comment (mario) [reply app/src/main/java/com/vanniktech/emoji/sample/MainActivity.java]: Yes.

File: gradle.properties [hunk 2]
```diff
@@ -14,4 +14,8 @@ POM_LICENCE_URL=http://www.apache.org/licenses/LICENSE-2.0.txt
 POM_LICENCE_DIST=repo
 
 POM_DEVELOPER_ID=vanniktech
-POM_DEVELOPER_NAME=Niklas Baudy
\ No newline at end of file
+POM_DEVELOPER_NAME=Niklas Baudy
+
+android.useAndroidX=true
+android.enableJetifier=true
```
Comment (vanniktech) [inline gradle.properties]: Can't we set this to false?

Comment (mario) [reply gradle.properties]: No. I mentioned it somewhere earlier that the library we use for ViewPager animations still didn't migrate to AndroidX so we need this.

Comment (vanniktech) [reply gradle.properties]: Ah right!

Comment (vanniktech) [reply gradle.properties]: Sorry, I forgot

Comment (mario) [reply gradle.properties]: No worries! :)

**Pull request merged**